articles/azure-large-instances/what-is-azure-large-instances.md

```diff
@@ -0,0 +1,194 @@
+---
+title: What is Azure for Large Instances?
+description: Provides an overview of Azure for Large Instances.
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.custom: references_regions
+ms.topic: conceptual
+ms.date: 06/01/2023
+---
+
+#  What is Azure Large Instances?
+
+Microsoft Azure offers a cloud infrastructure with a wide range of integrated cloud services to meet your business needs. 
+In some cases, though, you may need to run services without a virtualization layer.
+You may need root access and control over the operating system (OS).
+To meet this need, Azure offers Azure Large Instances (ALI) for several high-value, mission-critical applications.
+
+Azure Large Instances is comprised of dedicated large instances (compute instances). It features:
+
+* High-performance storage appropriate to the application (Fiber Channel).
+Storage can also be shared across ALI to enable features like scale-out clusters or high availability pairs with failed-node-fencing capability. 
+
+* A set of function-specific virtual LANs (VLANs) in an isolated environment.
+
+This environment also has special VLANs you can access if you're running virtual machines (VMs) on one or more Azure Virtual Networks (VNets) in your Azure subscription.
+The entire environment is represented as a resource group in that subscription.
+
+ALI is offered in over 30 SKUs from 2-socket to 24-socket servers and memory ranging from 1.5 TBs up to 24 TBs. 
+A large set of SKUs is also available with Optane memory.
+Azure offers the largest range of Azure Large Instances in a hyperscale cloud.
+
+## Why ALI?
```

Comment:
Can we use the abbreviated form Azure Large Instances instead? @ianmsgit 
```suggestion
## Why ALI?
```

Reply:
do you mean change the occurence in line 31? otherwise i don't see  change in line 33

Reply:
No wherever ALI is being used in text.Can we use Azure Large Instances (ALI) instead? @ianmsgit It is ok to use ALI in images where we have space constraints.But in the content lets use the full form wherever we can.

Reply:
@suzizuber : Tagging you to ensure this comment is addressed.